src/main/java/zeenea/connector/property/PropertyDefinition.java

```diff
@@ -66,18 +77,19 @@ public abstract class PropertyDefinition {
    *
    * @return the label of the property definition
    */
+  @Deprecated(since = "2.3.0", forRemoval = true)
```

Comment:
Idem ici : 
```suggestion
  
  @Deprecated(
      since =
          "Deprecated since version 2.3.0, use getName() instead. Scheduled for removal in version 3.0.0.",
      forRemoval = true)
```